haas/ext/switches/nexus.py

```diff
@@ -105,12 +105,6 @@ def connect(switch):
         console.expect('Password: ')
         console.sendline(switch.password)
 
-        console = pexpect.spawn('telnet ' + switch.hostname)
```

Comment:
`git-blame` blames me: dfdc7f8. Apparently I goofed somehow way back and nobody noticed until now. +1, merging.